examples/query.rs

```diff
@@ -0,0 +1,25 @@
+use mdns_sd::{ServiceDaemon, ServiceEvent};
```

Comment:
Could you please add a brief module doc comment at the beginning of this file? Something like:  (just a sample, you can change the wording/content).
```
//! A mDNS query client.
//! 
//! Run with:
//!
//! cargo run --example query <service_type>
//! 
```